ecosystem/sep-0006.md

```diff
@@ -820,9 +820,10 @@ Request Parameters:
-| `quote_id`                    | string                  | (optional) The `id` returned from a `SEP-38 POST /quote` response. If this parameter is provided and the user delivers the deposit funds to the Anchor before the quote expiration, the Anchor should respect the conversion rate agreed in that quote. If the values of `destination_asset`, `source_asset` and `amount` conflict with the ones used to create the [SEP-38] quote, this request should be rejected with a `400`.                                     |
 | `amount`                      | string                  | The amount of the `source_asset` the user would like to deposit to the anchor's off-chain account. This field may be necessary for the anchor to determine what KYC information is necessary to collect. Should be equals to `quote.sell_amount` if a `quote_id` was used.                                                                                                                                                                                            |
+| `funding_method`              | string                  | A method supported by the Anchor for transferring or settling assets. Must match one of the values specified in the corresponding `/info` response. This field is required to help the Anchor identify the necessary KYC information to collect.                                                                                                                                                                                                                      |
 | `account`                     | `G...` or `M...` string | The stellar or muxed account ID of the user that wants to deposit. This is where the asset token will be sent. Note that the account specified in this request could differ from the account authenticated via SEP-10.                                                                                                                                                                                                                                                |
+| `quote_id`                    | string                  | (optional) The `id` returned from a `SEP-38 POST /quote` response. If this parameter is provided and the user delivers the deposit funds to the Anchor before the quote expiration, the Anchor should respect the conversion rate agreed in that quote. If the values of `destination_asset`, `source_asset` and `amount` conflict with the ones used to create the [SEP-38] quote, this request should be rejected with a `400`.                                     |
```

Comment:
Can we add `funding_method` to the list of values that can't conflict with SEP-38 here?

Reply:
Is there such a list?

Reply:
> If the values of `destination_asset`, `source_asset` and `amount` conflict with the ones used to create the [SEP-38] quote, this request should be rejected with a `400`.